cloudify_cli/config/parser_config.py

```diff
@@ -244,26 +247,27 @@ def auto_generate_ids_argument():
 
 def parser_config():
     return {
-        'description': 'Manages Cloudify in different Cloud Environments',
+        'description': "Cloudify's Command Line Interface",
         'arguments': {
             '--version': {
-                'help': 'Show version information and exit',
+                'help': 'show version information and exit',
                 'action': cfy.version
             }
         },
         'commands': {
             'logs': {
-                'help': 'Handle Cloudify Manager logs',
+                'help': "Handle the Manager's logs",
                 'sub_commands': {
                     'get': {
```

Comment:
Compare `cfy blueprints download --output` and `cfy plugins download --output`, which, according to their help texts, seem to serve about the same purpose. So why here it's `logs --destination-path`?